### ISSUE TITLE
Building fails at OpenSSL dependency

### ISSUE THREAD

**Issue by cioraneanu** [WhisperSystems/Signal-iOS#2623]
### Bug description
Project fails while executing pod install part

### Steps to reproduce
- clone the repo
- execute `make dependencies`

**Actual result:** 
```
Pre-downloading: `OpenSSL` from `https://github.com/WhisperSystems/OpenSSL-Pod`, commit `b2d3c149102032a09aefbfb470885db4aa83efad`
[!] Failed to download 'OpenSSL'.
```

**Comment by benjuang** [comment]
I ran into this about a week ago.  Unfortunately, I didn't do a good job taking notes and tracking exactly what I did.  I'm going to ramble a bit as I read my bash history.  Disclaimer: I am not an iOS developer, came to see if I could maybe fix https://github.com/WhisperSystems/Signal-iOS/issues/2321 and have not had much luck.

I recall the error message being misleading - the Pre-downloading that CocoaPods does seems to do some post-processing after downloading and (at least for me), it was failing at the actual post-processing step.  You should probably try to do `git clone https://github.com/WhisperSystems/OpenSSL-Pod` to make sure downloading isn't actually an issue.  I tried doing `bundle exec pod install --verbose` to debug things, but the output was also not all that useful.  Verbose will display a temp directory (`/var/folders/....`) that contains some of the build data and a build log that provided much better information.

In the end, I discovered that somehow, despite installing and opening xcode, `xcodebuild -showsdks` said I had no sdks so when CocoaPods was trying to build OpenSSL-Pod, the missing SDKs caused it to die in ways it didn't expect.  The thing that (I think) fixed it for me was `sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer`.


TL;DR: Saw sometime similar, run `xcodebuild -showsdks` and see it lists SDKs. If not, run `sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer`.  Then try `make dependencies` again.  If it does, try `bundle exec pod install --verbose` and see what fails?

**Comment by michaelkirk** [comment]
@cioraneanu - can you provide some additional information?

macos version:
output of `xcode-select -p`
version of XCode you're using?
Output of  `bundle exec pod install --verbose`

**Comment by cioraneanu** [comment]
After pulling recent commits the issue seems to be gone.
